package.json

```diff
@@ -9,39 +9,41 @@
     "node": ">=8.5.0"
```

Comment:
Do you get an error with eslint?
`error eslint@5.12.1: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".`
Updating the node engine to ">=9.10.0" fixed this for me

Reply:
I don't think I saw that error. Worth just updating if that works though. Thanks.